### PR TITLE
engine: log k8s events to appropriate manifest log as well as global log [ch2644]

### PR DESCRIPTION
**Pull request by maiamcc**
Hello tbd,

Please review the following commits I made in branch maiamcc/event-in-mainfest-log:

66d790585c47b739ff76a218b2dfab83dcc206eb (2019-05-31 12:34:17 -0400)
store warn events on manifest state

37bcdbf7bb5eb595cc2c7a0fca51499f0ce24dc3 (2019-05-31 12:19:35 -0400)
test global log, only log warning events

d81501f1acbcfff3c2d1b2416448d5cda332153e (2019-05-31 12:07:40 -0400)
timestamp test

c6b0e531c678f8b602b0202f6d2019785a2ca8ba (2019-05-31 11:46:45 -0400)
tests wip

35be4adee1584ef92ad3de7f0eae04b1cc40fb0e (2019-05-30 16:52:25 -0400)
engine: log k8s events to appropriate manifest log as well as global log

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

**Comment by maiamcc** [comment]
Screen Shot 2019-05-31 at 12 52 24 PM

File: internal/engine/uid_map_manager_test.go [hunk 1]
```diff
@@ -59,6 +45,30 @@ func TestUIDMapManager(t *testing.T) {
 	f.assertObservedUIDUpdateActions(expectedAction)
 }
 
+func entityWithUID(t *testing.T, yaml string, uid string) k8s.K8sEntity {
+	es, err := k8s.ParseYAMLFromString(yaml)
+	if err != nil {
+		t.Fatalf("error parsing doggos yaml: %v", err)
```
Comment (jazzdan) [inline internal/engine/uid_map_manager_test.go]: Is this necessarily doggos YAML?

File: internal/engine/upper.go [hunk 2]
```diff
@@ -853,20 +853,23 @@ func handleServiceEvent(ctx context.Context, state *store.EngineState, action Se
...
 
-	if event.Type == "Warning" {
-		// TODO: log this to the manifest's log as well
-		logger.Get(ctx).Infof("event received for %s: %s:%s:%s: %s",
-			v.Manifest,
-			v.Entity.Name(),
-			v.Entity.Namespace(),
-			v.Entity.Kind,
-			event.Message)
+	if evt.Type != "Normal" {
```
Comment (jazzdan) [inline internal/engine/upper.go]: Nice!

File: internal/engine/upper.go [hunk 3]
```diff
@@ -853,20 +853,23 @@ func handleServiceEvent(ctx context.Context, state *store.EngineState, action Se
...
+		ms, ok := state.ManifestState(v.Manifest)
+		if !ok {
+			return
+		}
+
+		ms.CombinedLog = model.AppendLog(ms.CombinedLog, action, state.LogTimestamps)
+		logger.Get(ctx).Infof("%s%s", logPrefix(ms.Name.String()), action.MessageRaw())
```
Comment (jazzdan) [inline internal/engine/upper.go]: Is this supposed to be here?

Comment (maiamcc) [reply internal/engine/upper.go]: Yeahhhh it's how we get it into the global log :( unless you know a better way?

File: internal/engine/uid_map_manager_test.go (hunk 1 above)
Comment (maiamcc) [reply internal/engine/uid_map_manager_test.go]: ha no it's not!

**Pull request merged**